src/status_im/ui/screens/offline_messaging_settings/events.cljs

```diff
@@ -16,6 +18,45 @@
                               {:dispatch [:logout]}
                               (accounts-events/update-settings (assoc-in settings [:wnode network] wnode))))))
 
+(handlers/register-handler-fx
+ :save-wnode-transaction
+ (fn [{:keys [db] :as cofx} [_ wnode hash]]
+   (let [network     (ethereum/network->chain-keyword (:network db))
+         settings    (get-in db [:account/account :settings])
+         transaction (get-in db [:wallet :transactions hash])]
+     (when (not (nil? transaction))
```

Comment:
More idiomatic is probably `(when (seq transaction))` (e.g. https://stackoverflow.com/questions/3118962/what-is-the-correct-clojure-way-to-check-if-a-collection-is-non-empty)